tasks/browserSupport.js

```diff
@@ -2,14 +2,16 @@ const gulp = require('gulp');
 const config = require('./../config');
 
 gulp.task('browserSupport', function () {
+	const jsonParser = require('../lib/json-parser');
```

Comment:
Please require only those modules that are needed to run this task. As far as i see this package is needed only if browserSupport task is active. So this should be required inside the following if statement. 

Performance is getting more and more important while the project is growing and we already have performance issues. Requiring modules only if needed is the easiest way to save some microseconds...

Reply:
Good idea and I have implemented it as you suggested, but now we are requiring "json-parser" twice and have four lines of code that are duplicated in this file, once for browserSupport task and once for its watch task. For tasks like "browserSupport" that need to be watched, why don't we simply invoke watch from inside the task rather than creating a separate task for it and hence avoiding code duplication and unnecessary library loading?